roles/ansible-test/tasks/main.yaml

```diff
@@ -33,4 +33,13 @@
     chdir: "{{ _test_location }}"
     executable: /bin/bash
   environment: "{{ ansible_test_environment | default({}) }}"
-  shell: "source {{ ansible_test_venv_path }}/bin/activate; {{ ansible_test_executable }} {{ ansible_test_test_command }} {{ ansible_test_options }} --python {{ ansible_test_python }} -vvvv {{ _integration_targets }}"
+  shell: "source {{ ansible_test_venv_path }}/bin/activate; {{ ansible_test_executable }} {{ ansible_test_test_command }} {{ ansible_test_options }} --python {{ ansible_test_python }} -vvvv {{ _integration_targets }}; {{ ansible_coverage_report }}"
```

Comment:
I feel the generate coverage report will have to move into a post-run playbook, that is more inline with other log collection jobs.  We may have to hard code paths a little, to work around the ansible role issue you mentioned before.

Reply:
Just to confirm, this should not need to be done, if we move ansible_coverage_report to post-run phase.